applications/pusher/src/modules/pm_apple.erl

```diff
@@ -61,20 +69,24 @@ code_change(_OldVsn, State, _Extra) ->
 maybe_send_push_notification('undefined', _) -> 'ok';
 maybe_send_push_notification(Pid, JObj) ->
     TokenID = kz_json:get_value(<<"Token-ID">>, JObj),
-    Sender = kz_json:get_value(<<"Alert-Body">>, JObj),
-    CallId = kz_json:get_value(<<"Call-ID">>, JObj),
-    APNsTopic = apns_topic(JObj),
-    apns:push_notification(Pid
-                          ,TokenID
-                          ,#{aps => #{alert => #{'loc-key' => <<"IC_MSG">>
-                                                ,'loc-args' => [Sender]
-                                                }
-                                     ,sound => <<"ring.caf">>
-                                     }
-                            ,'call-id' => CallId
-                            }
-                          ,#{apns_topic => APNsTopic}
-                          ).
+    Topic = apns_topic(JObj),
+    TopicArg = #{apns_topic => Topic},
+    Msg = build_payload(JObj),
+    lager:debug_unsafe("pushing topic ~s for token-id ~s : ~s", [Topic, TokenID, kz_json:encode(kz_json:from_map(Msg), ['pretty'])]),
```

Comment:
why the debug_unsafe ?

Reply:
@mark2600 for proper logging of json. lager usually truncs it